api/bundle/client.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package bundle provides access to the bundle api facade.
+// This facade contains api calls that are specific to bundles.
+
+package bundle
+
+import (
+	"github.com/juju/errors"
+	"github.com/juju/loggo"
+
+	"github.com/juju/juju/api/base"
+	"github.com/juju/juju/apiserver/params"
+)
+
+var logger = loggo.GetLogger("juju.api.bundle")
+
+// Client allows access to the bundle API end point.
+type Client struct {
+	base.ClientFacade
+	st     base.APICallCloser
```

Comment:
I wonder if you actually need it? Since you are not using it anywhere except in the constructor to create the client facade instance...

Reply:
u r correct. i left this because may be required for signature used in all client facades.